mkdocs.yaml

```
@@ -1,9 +1,9 @@
 site_name: Autoware Universe Documentation
 site_url: https://autowarefoundation.github.io/autoware.universe
 repo_url: https://github.com/autowarefoundation/autoware.universe
-edit_uri: edit/main/
+edit_uri: https://github.com/autowarefoundation/autoware.universe/edit/main/docs/
```

@satoshi-ota Sorry, `docs` was unnecessary.

```suggestion
edit_uri: https://github.com/autowarefoundation/autoware.universe/edit/main/
```

---

fixed.